modules/agentless-s3-bucket/variables.tf

```diff
@@ -0,0 +1,26 @@
+
+variable "tags" {
+  description = "A map of additional tags to add to the IAM role/profile created"
+  type        = map(string)
+  default     = {}
+}
+
+variable "iam_delegate_role_arn" {
+  description = "Reference ARN to the Agentless delegate role"
+  type        = string
+}
+
+variable "iam_rds_assume_role_arn" {
+  description = "Reference ARN to the Agentless RDS assume role"
+  type        = string
+}
+
+variable "primary_kms_key_arn" {
+  description = "Primary KMS key ARN to encrypt the exported data"
+  type        = string
+}
+
+variable "primary_kms_key_region" {
```

Comment:
The pattern is usually to encrypt with a key in the same region, do you think we need to support this?

Reply:
We do use a key for each region. I need this variable to check whether this bucket is being created in the same region as the primary key, because in that case we don't need to create a replica.